io.github.leeroymanea.synology-note-station.yaml

```diff
@@ -0,0 +1,42 @@
+# copied/inspired from https://github.com/flathub/com.synology.SynologyDrive/blob/master/com.synology.SynologyDrive.yaml
+id: io.github.leeroymanea.synology-note-station
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: launch.sh
+finish-args:
+  # needed for launch.sh (code from synology)
+  - --filesystem=home
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+modules:
+  - name: synology-note-station
+    buildsystem: simple
+    sources:
+      - type: file
+        sha256: a4c763bddda21bd211b0c8afdcff7e72a3edddc2392470fec8ce8123a0531751
+        url: "https://global.synologydownload.com/download/Utility/NoteStationClient/2.2.4-703/Ubuntu/x86_64/synology-note-station-client-2.2.4-703-linux-x64.deb"        
+      - type: file
+        path: io.github.leeroymanea.synology-note-station.metainfo.xml
+    build-commands:
+       # unzip folder
+      - ar x synology-note-station-client-2.2.4-703-linux-x64.deb
+      - tar -xf data.tar.xz
+      # move to app location
+      - mv usr/share /app/share
+      - mv opt /app/opt
+      # rename some files
+      - mv /app/share/applications/synology-note-station-client.desktop /app/share/applications/io.github.leeroymanea.synology-note-station.desktop
+      - find /app/share/icons/hicolor -type f -execdir sh -c 'mv "$0" io.github.leeroymanea.synology-note-station.png' {} \;
```

Comment:
there is `rename-icon` that can be used in the manifest for that purpose. and it fixes the desktop file at the same time.

Reply:
cannot execute this commands:
```
rename-desktop-file: command not found
rename-icon: command not found
```